thorlcr/graph/thgraphmaster.cpp

```diff
@@ -2760,7 +2765,13 @@ void CMasterGraph::getFinalProgress()
                 if (s>=n)
                     break;
             }
-            throw MakeGraphException(this, 0, "Timeout receiving final progress from slaves - these slaves failed to respond: %s", slaveList.str());
+            if (aborting)
+            {
+                OERRLOG("Timeout receiving final progress from slaves - these slaves failed to respond: %s", slaveList.str());
```

Comment:
this should be changed to a WARNLOG (shouldn't be an operator error), because if 'aborting' then there will have been proceeding reported errors and this is a side-effect.